airflow/providers/elasticsearch/provider.yaml

```diff
@@ -23,7 +23,6 @@ description: |
 
 suspended: false
 versions:
-  - 5.0.0
```

Comment:
I am torn. I am in a position (and always been doing that - that the version of the provider should be updated by the Release Manager during document preparation - so my recommendation so far was that users who are adding significant changes just add it at the top of the file and then I'd update those when preparing the docs - but I saw that @eladkal asks to add those versions for breaking changes.

I think thise might be a bit confusing - because it might lead for example to - unknowingly if you are not careful and don't look when  - to bump the version twice (so I opted for only adding the version once by RM when releasing - giving a clear split of responsibility - contributor just explains what was done, but release manager bumps the version). But it seems to work for @eladkal to bump those versions by contributors, before the release is made, so yeah, I can add it as well. No problem for me.

Reply:
Added it back.

Reply:
yes, sorry for the double work. Wished to know & discuss when to add and when not to add and if we have a rule here :)

Reply:
No worries at all - I am all for iterating and improving such changes even if it means a number of push/fixup/cycles. This is the whole point of reviews, the more eyeballs asking the right questions it's also more mistakes avoided at the earliest possible time. 

Any iteration at the moment PR is being reviewed costs just an update + push. when thing gets  broken after merge, it gets more messy as more people might feel the impact (multiple contributions) then we have another point in this case which is releasing providers and there we have RCs and potentailly finding problems. But really costly is when any kind of mistake gets released and already used by the users.

So by all means - the more comments and iteration at the early PR stage, the less costly it is in fact.